### PR TITLE
<Text/> - change disabled+light color

### PR DESCRIPTION
**Pull request by gilmeir**
#3859

### CODE REVIEW

**Comment by gilmeir** [comment]
@LittleTal 
There are a few states missing in the Zeplin spec, so I'm listing them down here.
It refers only to `disabled` skin:
regular - D10
secondary - D20
light - D80-70
light + secondary - D80-70

**Comment by LittleTal** [comment]
@gilmeir There are only 2 types of disabled, dark and light (primary and secondary on both cases should be the same) , I've updated [here](https://zpl.io/aNmZW5Q) the disabled for light

File: src/Text/Text.st.css [hunk 1]
```diff
@@ -73,11 +73,15 @@
 }
 
 .root:skin(disabled),
-.root:skin(disabled):light,
 .root:skin(disabled):secondary {
   color: value(D10-30);
 }
 
+.root:skin(disabled):light,
+.root:skin(disabled):light:secondary {
```
Comment (gilmeir) [inline src/Text/Text.st.css]: Do I really need to add the :secondary selector? it's for both true/false

File: src/Text/test/Text.spec.js [hunk 2]
```diff
@@ -85,17 +85,6 @@ describe('Text', () => {
         const { driver } = render(<Text light>Hello</Text>);
         expect(await driver.isLight()).toBe(true);
       });
-
```
Comment (gilmeir) [inline src/Text/test/Text.spec.js]: Currently the component doesn't propagate `light` as is to the st.css.
I've changed that, `light` currently propagated as is,
and whether/how to style it - it's in the css

Comment (gilmeir) [reply src/Text/test/Text.spec.js]: Also, we don't want to maintain visual testing as part as unit tests,
and we already have a `text variations` visual test to test all the permutations.

File: src/Text/test/Text.visual.js [hunk 3]
```diff
@@ -74,7 +74,13 @@ storiesOf('Text/Styles', module).add('styles variations', () => (
             {Object.keys(SKINS).map(skin =>
               [false, true].map(light =>
                 [false, true].map(secondary => (
-                  <span style={{ margin: '12px' }}>
+                  <span
+                    style={{
+                      margin: '12px',
+                      backgroundColor:
+                        light && skin === 'disabled' ? 'black' : 'transparent',
```
Comment (gilmeir) [inline src/Text/test/Text.visual.js]: dark background for light colors

File: src/Text/Text.st.css [hunk 4]
```diff
@@ -77,8 +77,7 @@
   color: value(D10-30);
 }
 
-.root:skin(disabled):light,
-.root:skin(disabled):light:secondary {
+.root:skin(disabled):light {
```
Comment (gilmeir) [inline src/Text/Text.st.css]: @mosheKerbel 
it was unnecessary. 
merging.

**Pull request merged**